### PR TITLE
attempt to get travis cron job to only run translations

### PR DESCRIPTION
**Pull request by benjiwheeler**
### Resolves:

Resolves https://github.com/LLK/scratch-www/issues/3215 , to correct a mistake in https://github.com/LLK/scratch-www/pull/3643 .

### Changes:

Second attempt to configure .travis.yml file to cause Travis's develop branch cron job to trigger our translation push script.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -135,13 +136,10 @@ jobs:
         - master
   - stage: smoke
     script: npm run test:integration:remote
+  - stage: update translations
```
Comment (chrisgarrity) [inline .travis.yml]: After re-reading the [conditional builds, jobs, and stages doc](https://docs.travis-ci.com/user/conditional-builds-stages-jobs/) I think we're mixing up jobs and stages here.
If we keep update translations as a stage, then to be consistent, this part should only define the script, and the condition should be in the stages section below.

File: .travis.yml [hunk 2]
```diff
@@ -135,13 +136,10 @@ jobs:
...
+    script: npm run i18n:push
+    if: branch == develop AND type == cron
 stages:
 - test
 - name: smoke
   if: type != pull_request
```
Comment (chrisgarrity) [inline .travis.yml]: I think this is going to run the smoke test on every `cron` job unless we change it to `type NOT IN (cron, pull_request`

Comment (benjiwheeler) [reply .travis.yml]: OK, take a look now

File: .travis.yml (hunk 1 above)
Comment (benjiwheeler) [reply .travis.yml]: OK, take a look now

**Pull request merged**